collect_app/src/main/java/org/odk/collect/android/widgets/interfaces/ImageWidget.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2017 Nafundi
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.odk.collect.android.widgets.interfaces;
+
+public interface ImageWidget extends FileWidget {
```

Comment:
Maybe call this something else @grzesiek2010 since there's another ImageWidget class already. BaseImageWidget?

Reply:
I've just added an "interface" word for all interfaces.